tests/tests_cpp/tests_utils.cpp

```diff
@@ -0,0 +1,36 @@
+#include <catch2/catch_test_macros.hpp>
+#include <iostream>
+
+#include "msg_conversions.h"
+
+TEST_CASE("Multiarray Testing", "[utils][msgs]") {
+    for (int l = 0; l < 10; l++) {
```

Comment:
can you add a clarification comment here? You're running this test 10 times for an array with 5 dimensions, right?